.github/CODEOWNERS

```diff
@@ -7,7 +7,11 @@
 # ASM Team
 /appsec/ @DataDog/asm-php
 
+# APM libdatadog
+/components-rs/ @Datadog/libdatadog-apm
+compile_rust.sh @Datadog/libdatadog-apm
+
 # Release files
-Cargo.lock @DataDog/apm-php @DataDog/profiling-php
-package.xml @DataDog/apm-php @DataDog/profiling-php @Datadog/asm-php
-VERSION @DataDog/apm-php @DataDog/profiling-php @Datadog/asm-php
+Cargo.lock @DataDog/apm-php @DataDog/profiling-php @Datadog/libdatadog-apm
+package.xml @DataDog/apm-php @DataDog/profiling-php @Datadog/asm-php @Datadog/libdatadog-apm
+VERSION @DataDog/apm-php @DataDog/profiling-php @Datadog/asm-php @Datadog/libdatadog-apm
```

Comment:
I don't think you're going to release versions, so these are unnecessary.
Everything else is fine.